ohsome_quality_analyst/indicators/base.py

```diff
@@ -138,3 +143,17 @@ def _get_default_figure(self) -> None:
         raw = fig.to_dict()
         raw["layout"].pop("template")  # remove boilerplate
         self.result.figure = raw
+
+    def get_template(self) -> None:
+        """Get template for indicator."""
+        indicator_key = camel_to_snake(type(self).__name__)
+        directory = get_module_dir(
+            "ohsome_quality_analyst.indicators.{}".format(indicator_key)
+        )
+        file = glob.glob(directory + "/metadata.yaml", recursive=True)
```

Comment:
```suggestion
        file = os.path.join(directory, "/metadata.yaml")
```
Should be enough here. Or is it not?